src/auditlog/models.py

```diff
@@ -62,6 +63,60 @@ def log_create(self, instance, **kwargs):
             return self.create(**kwargs) if db is None or db == '' else self.using(db).create(**kwargs)
         return None
 
+    def log_m2m_changes(self, changed_queryset, instance, action, **kwargs):
+        """
+
+        Helper method to create a new log entry. This method automatically populates some fields when no explicit value
+        is given.
+
+        :param instance: The model instance to log a change for.
+        :type instance: Model
+        :param action: type of action.
+        :type instance: int LogEntry.ACTIONS choices
+        :param kwargs: Field overrides for the :py:class:`LogEntry` object.
+        :return: The new log entry or `None` if there were no changes.
+        :rtype: LogEntry
+        """
+
+        pk = self._get_pk_value(instance)
+        if changed_queryset is not None:
+            kwargs.setdefault('content_type', ContentType.objects.get_for_model(instance))
+            kwargs.setdefault('object_pk', pk)
+            kwargs.setdefault('object_repr', smart_text(instance))
+
+            if isinstance(pk, integer_types):
+                kwargs.setdefault('object_id', pk)
+
+            get_additional_data = getattr(instance, 'get_additional_data', None)
+            if callable(get_additional_data):
+                kwargs.setdefault('additional_data', get_additional_data())
+
+            kwargs.setdefault('action', action)
+
+            instance_verbose_name = str(smart_text(instance._meta.verbose_name))
```

Comment:
This is the wrong object to use. Let's say there is the following model:
```python
class Tag(Model):
    name = CharField(...)

class User(Model):
     random_tags = ManyToManyField(Tag)
```
When the tags of a user change, `instance` here is the instance of `User`, so `instance_verbose_name` is `"user"`, which results in the following log entry:
```json
{"changes": {"user": ["Added", ["1 - tag1", "2 - tag2"]]}}
```
This is clearly wrong, as one would expect to see `"random_tags"` is the key in the changes dict here.